projects/ontimize-web-ngx/src/lib/interfaces/o-table-onclick.interface.ts

```diff
@@ -3,5 +3,10 @@ export interface OnClickTableEvent {
   row: any;
   /** row index */
   rowIndex: number;
+  /** mouseEvent */
   mouseEvent: MouseEvent;
+  /**attr column   */
+  columnId: string;
```

Comment:
We don't use the name `columnId` anywhere in the code. I think`column` or `columnName` are more suitable names for this parameter.